e2e_samples/mdw_governance/README.md

```diff
@@ -1,6 +1,8 @@
 # Modern Data Warehouse Data Governance end-to-end deployment with Purview <!-- omit in toc -->
 
-This sample demonstrates how to provision end-to-end modern data warehouse solution using [Azure DevOps pipelines](https://azure.microsoft.com/en-au/services/devops/pipelines/) to deploy across Dev, Test and Prod environments. It particularly focuses on data governance, using [Azure Purview](https://azure.microsoft.com/en-au/services/purview/), to track data assets and lineage to allow data managers to understand the quality and source of the data, as well as to understand if there is PII and other sensitive information in your data estate.
+This sample demonstrates how to provision end-to-end modern data warehouse solution using Azure DevOps pipelines to deploy Dev, Test and Prod environments.
+
+The sample focuses on data governance, using Purview, to track data assets and lineage to allow data managers to understand the quality and source of the data, as well as to understand if there is PII etc. in the data estate. It also uses [presidio](https://github.com/microsoft/presidio), to anonymize parts of the scanned data and redcue the potential to expose PII. 
```

Comment:
This seems to completely replace the above paragraph. Can you append the last sentence "It also uses presidio..." to the end of the paragraph instead? Also typo: reduce*